runner.go

```diff
@@ -108,10 +108,13 @@ func (r *Runtime) groupTaskAndFlagArgs(args []string) map[string][]string {
 				currFlagsList = []string{}
 			}
 
-			// If the arg is a valid task, start tracking the taks/flag group.
+			// If the arg is a valid task, start tracking the task/flag group.
 			if _, ok := r.registry.definitions[arg]; ok {
 				currTaskName = arg
 				currFlagsList = []string{}
+			} else {
+				log.Printf("Unrecognized task: %s, ignoring it and following flags\n", arg)
+				// Note that in this case, currTaskName must be "" to ignore following flags.
```

Comment:
Hmm are we saying that this will only happen if we haven't found a task in the args yet?

Reply:
Yeah, that the earlier block where we parse flags checks `currTaskname == ""` and ignores a flag if so. Just left a note because the relation is not explicit.